vis.mac

```diff
@@ -78,6 +78,8 @@
 #/DarkRate/SetConvert 1.119 #For HPDs
 #/DarkRate/SetConvert 1.126 #For Box and Line PMTs
 
+##Comment out lines the create, open OGLSX, zoom, viewpointThetaPhi, and upVector lines below if using RayTracer
```

Comment:
This comment is left over from the last commit. Remove it and make sure vis.mac doesn't show up in the diff since I don't think anything is changed here anymore.